src/pal/tools/gen-buildsys-clang.sh

```diff
@@ -108,6 +108,8 @@ llvm_link="$(locate_llvm_exec link)"
 [[ $? -eq 0 ]] || { echo "Unable to locate llvm-link"; exit 1; }
 llvm_nm="$(locate_llvm_exec nm)"
 [[ $? -eq 0 ]] || { echo "Unable to locate llvm-nm"; exit 1; }
+llvm_ranlib="$(locate_llvm_exec ranlib)"
```

Comment:
My surface understanding standing is that llvm-ranlib is intended to handle the clang output when when using the `-flto` flag, whereas the standard ranlib does not. Really, I just hit errors with using the standard ranlib when trying to do this and looking around online indicated that llvm-ranlib should be used in order to use LTO.

Reply:
It was removed in https://github.com/dotnet/coreclr/pull/1363 because openSUSE distro did not have ranlib. If openSUSE is not exercised in the CI, that might be broken.

Reply:
@am11 thank you for the info. I had a feeling that it was removed intentionally in the past, but I haven't remembered the reason. We don't build anything on OpenSUSE in our CI.